html-parser/build.gradle.kts

```diff
@@ -1,19 +1,44 @@
 plugins {
-    buildsrc.convention.`kotlin-jvm`
+    buildsrc.convention.`kotlin-multiplatform`
 }
 
-java {
-    registerFeature("jsExecution") {
-        usingSourceSet(sourceSets["main"])
-    }
-}
+//java {
+//    registerFeature("jsExecution") {
+//        usingSourceSet(sourceSets["main"])
+//    }
+//}
+//
+//dependencies {
+//    implementation(projects.dsl)
+//    api(projects.fetcher.baseFetcher)
+//    api(Deps.jsoup)
+//
+//    "jsExecutionImplementation"(projects.fetcher.browserFetcher)
+//
+//    testImplementation(projects.testUtils)
+//}
```

Comment:
TODO clean this up

Reply:
I think the simplest replacement for the jsExecution feature variant is to just publish the browser-fetcher library, just like how http-fetcher is published. I think this will have the same outcome, and it will be very easy to do, when the publish-kotlin-multiplatform buildSrc convention plugin is working

What do you think @christian-draeger? Does it make sense to you?